std/async/delay.ts

```diff
@@ -1,9 +1,7 @@
 // Copyright 2018-2020 the Deno authors. All rights reserved. MIT license.
 /* Resolves after the given number of milliseconds. */
-export function delay(ms: number): Promise<void> {
-  return new Promise((res): number =>
-    setTimeout((): void => {
-      res();
-    }, ms)
-  );
+export const delay = (ms: number): Promise<void> => {
```

Comment:
Again, the style guide clearly disallows this: https://deno.land/manual/contributing/style_guide#top-level-functions-should-not-use-arrow-syntax. If you want to know why, you're welcome to ask in the discord: https://discord.gg/deno _before_ opening lots of PRs.